topics/icrc_25_signer_interaction_standard.md

```diff
@@ -73,13 +74,28 @@ The purpose of the `permission` messages is to grant the relying party access to
 
 While processing the request from the relying party, the signer can cancel it at any time by sending an error in response.
 
-`errorType` (`text`): The reason behind the cancellation. Possible values:
-- `"VERSION_NOT_SUPPORTED`: The version of the standard is not supported by the signer.
-- `"NETWORK_NOT_SUPPORTED"`: The network on which the action was requested is not supported by the signer.
-- `"NOT_GRANTED"`: The signer has not granted permission to perform the action.
-- `"UNKNOWN"`: The reason is unknown.
+The error is an object comprising the `code`, `message` and optional `data` fields as described in the [JSON-RPC 2.0 Specification](https://www.jsonrpc.org/specification#error_object). In addition the the pre-defined errors, the following values are supported:
```

Comment:
I would make an `##Errors` category, and have the full table of all errors there. And then only list `possible errors: 11000, 12002, ...` on the message itself.